kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java

```diff
@@ -332,6 +335,18 @@ public static ConfigDef newConfigDef() {
           Width.LONG,
           "S3 Proxy Password"
       );
+
+      configDef.define(
+          SSE_AWS_KMS_ID_CONFIG,
+          Type.STRING,
+          SSE_AWS_KMS_ID_DEFAULT,
+          Importance.LOW,
+          "The S3 Server Side Encryption KMS custom id/alias name.",
```

Comment:
Have you thought about using an enumeration so that the configdef describes the allowable values?

Also, it would probably be more beneficial to have a more clear description for those that don't know offhand what this entails, such as "S3 Server Side Encryption Key" for the title and for the description something like: "The name of the AWS Key Management Service (AWS-KMS) key to be used for server side encryption of the S3 objects. No encryption is used when no key is provided, but it is enabled when `aws:kms` is specified."

Also, when adding or changing any configuration properties we should also update the `kafka-connect-s3/docs/configuration_options.rst` file in this PR.